src/org/rascalmpl/values/RascalValueFactory.java

```diff
@@ -152,6 +152,8 @@ public class RascalValueFactory extends AbstractValueFactoryAdapter implements I
 	public static final Type Production_Priority = tf.constructor(uptr, Production, "priority", Symbol, "def", tf.listType(Production), "choices");
 	public static final Type Production_Composition = tf.constructor(uptr,  Production,  "composition", Production, "lhs", Production, "rhs");
 	public static final Type Production_Associativity = tf.constructor(uptr, Production, "associativity", Symbol, "def", Associativity, "assoc", tf.setType(Production), "alternatives");
+	public static final Type Production_Error = tf.constructor(uptr, Production, "error", Symbol, "def", Production, "prod", tf.integerType(), "dot");
+	public static final Type Production_Skipped = tf.constructor(uptr, Production, "skipped", Symbol, "def");
```

Comment:
shouldn't we also be building special IValues here? like those that exist for `appl` and friends?

Reply:
I will discuss this with Jurgen.